### ISSUE TITLE
Unknown transformation name es6 after upgrading to 1.4.0

### ISSUE THREAD

**Issue by z6aak** [kiwiupover/ember-cli-spinjs#155]
After upgrading to 1.4.0 and running ember build, I'm having this issue: 

`while import node_modules/spin.js/spin.js: found an unknown transformation name es6. Available transformNames are: amd`

**Comment by cafreeman** [comment]
Ditto here.

Here's my version info:

```
ember-cli: 2.16.2
node: 8.9.3
os: darwin x64
```

**Comment by adriaanvanrossum** [comment]
Same here. Also had this before ember-cli-spinjs version 1.4.0

```
ember-cli: 2.18.2
node: 8.9.4
os: darwin x64
```

**Comment by kiwiupover** [comment]
I will have a look at this thanks for reporting this.

**Comment by kiwiupover** [comment]
closed by #160
